### DISCUSSION TITLE
TI Mindmap - Sep 2024 - When the new source code will be available?

### DISCUSSION THREAD

**Discussion by simillo01** [format81/TI-Mindmap-GPT#14] [General]
Congratulations on the new release! I am very excited and can't wait to try the new version of ti-mindmap. When will the code be available? I discovered this project a few days ago and I fell in love with it. I would like to test it and report bugs as well as propose new features. I also found some bugs in the previous version but maybe they are no longer present in the new version! ;)

**Comment by format81** [comment]
Hi, I'm glad you like the project. The last changes were made a few days ago. Please report any bugs you found. Contributions are always welcome. We have a roadmap but we are open to new ideas. 
Thanks
